2017, new year post

mateoclarke opened the pull request:
Can I get some 👀  on this?

I would like to get this published by the end of the week and included in our next newsletter.

cc/ @mscarey @VictoriaODell @amaliebarras @mapineda

mateoclarke committed:
2017, new year post

mateoclarke committed:
small thing

mapineda commented:
Looks great! Amazing accomplishments for 2016 and the years prior! Excited to see the impact going forward.  - We'll include a link to this in the newsletter...which we should send out soon... what are 3-5 points we should include in the newsletter from this post?

amaliebarras committed:
fixing apply link.

looks great dude. I like the flow here are some of our successes, here are some challenges ahead, here's how you can get involved. also solid job of framing some of the problems our coreteam will solve this year. 

fixed a small markdown link thing.

mscarey committed:
Update 2017-01-17-yearly-review.md

mscarey commented:
Good! My only real concern is whether it's too early to debut the name "Open Austin Labs" without a longer blog post explaining what that is and what the roadmap for it will be. You could say more generally that the third priority is to have certain projects that get selected because they meet the criteria of "external stakeholders, clear goals and deadlines, and potential for funding," and to create an organizational commitment not just to providing support for those projects, but actually ensuring that they get delivered. (If that's a fair description.) The heading could be "Priority Projects". Or "Flagship Projects".

mateoclarke committed:
second draft

mateoclarke committed:
resolved merge
